app/renderer/components/Session/NewSessionForm.js

```diff
@@ -98,7 +99,17 @@ export default class NewSessionForm extends Component {
               </Col>
             </Row>)}
             <Row>
-              <Col span={24}>
+              <Col span={22}>
+                <FormItem>
+                  <Checkbox
+                    onChange={(e) => setAddVendorPrefixes(e.target.checked)}
+                    checked={addVendorPrefixes}
+                  >
+                    {t('Automatically add necessary Appium vendor prefixes on start')}
```

Comment:
I assume this string should be also added into `en` resources

Reply:
https://github.com/appium/appium-desktop/pull/1859